.github/workflows/stale_issues.yml

```diff
@@ -0,0 +1,45 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v3
+      with:
+        # Setting messages to an empty string will cause the automation to skip
+        # that category
+        ancient-issue-message: Greetings!! We noticed that this issue is very old and has not been updated in a very long time. We will close this issue for now. Please feel free to reach out and open a new issue if you think this is still a relevant issue or if it is an error.
```

Comment:
Is this the language we are using cross SDKs?  I'd lean towards removing some of the less matter of fact/precise language like "Greetings!!" (this is an automated response, two exclamation marks feels a little overly enthusiastic).  and "very old" (imprecise).  and "for now" (not straight forward - does this mean we have a process to re-open these?)

Reply:
I'm assuming it is cross SDK regardless..

Reply:
https://github.com/aws/aws-sdk-js/pull/3242

Reply:
This is not cross SDK